components/speedreader/rust/lib/src/speedreader.rs

```diff
@@ -194,8 +197,7 @@ impl SpeedReader {
                     declarative_rewrite: Some(_),
                     ..
                 }) => RewriterType::Streaming,
-                Some(_) => RewriterType::Heuristics,
-                None => RewriterType::Unknown,
+                _ => RewriterType::Unknown,
             }
         } else {
             RewriterType::Unknown
```

Comment:
yeah, all this code is over-abstracted. looking forward to deleting it as soon as possible. i renamed it to `get_rewriter_type_from_list(..)`, and since neither Heuristics or Readability use lists, the new name makes more sense for them not to be returned.